src/contrasts.jl

```diff
@@ -224,6 +266,17 @@ regression model, dummy coding leads to an intercept that is the mean of the
 dependent variable for base level.
 
 Also known as "treatment coding" (`contr.treatment` in R) or "one-hot encoding".
```

Comment:
I'm not sure about dropping mention of R. On the one hand, I'm sympathetic to the fact that we can't realistically mention every single alternative. On the other hand, I suspect that many users will be coming from R, but that might be biased by my own experience :)